cluster-api/src/main/java/io/aiven/klaw/clusterapi/services/SchemaService.java

```diff
@@ -50,30 +50,39 @@ public SchemaService(ClusterApiUtils clusterApiUtils) {
     this.clusterApiUtils = clusterApiUtils;
   }
 
+  /*
+  If force register is enabled
+    1. Get subject compatibility in a var 'x'
+    2. Set subject compatibility to NONE
+    3. Register schema
+    4. If x (subject compatibility) == NOT_SET, get global compatibility
+    5. Apply global compatibility on subject level
+  If force register is not enabled
+    1. Register schema
```

Comment:
Personally I think this kind of doc is not really useful as it's a non-compiled way to define what the function does. And for example, point 2. is not correct according to the implementation. It should be: 
> 2. Set subject compatibility to NONE if it's not already NONE.

If the code is too complicated I would personally extract those points 1 to 5 in functions with a descriptive name.

Reply:
Moved the description to controller call